jujugui/static/gui/src/app/components/unit-details/test-unit-details.js

```diff
@@ -59,19 +59,48 @@ describe('UnitDetails', function() {
         unit={fakeUnit} />);
 
     assert.deepEqual(output.props.children[0],
```

Comment:
It's unlikely, the call signature for deepEquals is 
`assert.deepEqual(actual, expected, message)`
You're comparing `output.props.children[0]` to itself and then have some jsx as the message if it's not